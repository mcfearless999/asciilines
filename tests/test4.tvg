10 8
A 1 2 h 4
B 5 2 v 6
C 2 6 h 12
D 2 3 v 2
