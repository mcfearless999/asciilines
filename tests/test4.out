........
..AAAA..
...D..CC
...D....
........
..B.....
..B.....
..B.....
..B.....
..B.....
